src/api/track.ts

```diff
@@ -115,18 +109,35 @@ class TrackAPI {
         },
       });
 
-      const header = JSON.stringify({
-        alg: 'HS256',
-        typ: 'JWT',
-      });
-      const payload = JSON.stringify({
+      let sclVal = -1;
+      let cslVal = -1;
+      try {
+        const [sclRes, csl] = await Promise.all([
+          Http.request({
+            host: `https://${pingHost}`,
+            method: 'GET',
+            path: 'ping',
+          }),
+          ping(`wss://${pingHost}`),
+        ]);
+        const { scl }: any = sclRes;
+        sclVal = scl;
+        cslVal = csl;
+      } catch (err) {
+        
```

Comment:
Could do a `Logger.warn` here

Reply:
Rather be somewhat opaque at this step, and we're logging errors server-side

Reply:
Maybe add a comment `// do nothing`, the empty block just looks sorta odd (though obvious what its doing)

Reply:
Done